File: C#/Iniciante/Pagina 2/1065.cs
using System;

//Pares entre Cinco Números

//Faça um programa que leia 5 valores inteiros. Conte quantos destes valores digitados são pares e mostre esta informação.

//Entrada
//O arquivo de entrada contém 5 valores inteiros quaisquer.

//Saída
//Imprima a mensagem conforme o exemplo fornecido, indicando a quantidade de valores pares lidos.

class URI
{
    static void Main(string[] args)
    {
        int quantity = 0;
        for (int i = 0; i < 5; i++)
        {
            int number = int.Parse(Console.ReadLine());
            if (number % 2 == 0)
            {
                quantity++;
            }
        }

        Console.WriteLine($"{quantity} valores pares");
    }

}

File: Python/Iniciante/Pagina 2/1065.py
# Pares entre Cinco Números

# Faça um programa que leia 5 valores inteiros. Conte quantos destes valores digitados são 
# pares e mostre esta informação.

# Entrada
# O arquivo de entrada contém 5 valores inteiros quaisquer.

# Saída
# Imprima a mensagem conforme o exemplo fornecido, indicando a quantidade de valores pares
# lidos.
pair_numbers = []

for _ in range(5):
  number_input = int(input())
  if number_input % 2 == 0:
    pair_numbers.append(number_input)
    
print(f"{len(pair_numbers)} valores pares")